brambling/views/core.py

```diff
@@ -57,6 +58,10 @@ def get_context_data(self):
                 order.event = re_dict[order.event_id]
                 order.event.order = order
 
+            # Exclude registered events
+            upcoming_events_interest = upcoming_events_interest.exclude(pk__in=registered_events_qs)
+            upcoming_events = upcoming_events.exclude(pk__in=registered_events_qs)
```

Comment:
should this use the list version instead, so that it doesn't use a subquery, or do we not need to worry about that because Postgres?